ports/kfr/vcpkg.json

```
@@ -1,10 +1,10 @@
 {
   "name": "kfr",
-  "version-semver": "6.0.2",
+  "version-semver": "6.0.3",
   "description": "Fast, modern C++ DSP framework.",
   "homepage": "https://www.kfr.dev/",
   "license": "GPL-2.0",
```

```suggestion
  "license": "GPL-2.0-or-later",
```
Cf: https://github.com/lighttransport/tinyusdz?tab=readme-ov-file#third-party-licenses  
Commercial licensing also exists. This cannot be expressed here, but it could be expressed in the comment in the `copyright` file.)

---

Thanks for your reminder and suggestion, it has been updated.